app/views/fields/has_many/_show.html.erb

```diff
@@ -19,15 +19,17 @@ from the associated resource class's dashboard.
 %>
 
 <% if field.resources.any? %>
+  <% order = field.order_from_params(params.fetch(field.name, {})) %>
+  <% page = params.fetch(field.name, {}).fetch(:page, nil) %>
```

Comment:
Hm, ideally we shouldn't be using `params` here, but instead be something that we pass down from the controller in some way. Here we would access them, eg. as properties of `field`.

But then again, it's not really your change that is doing that, as we were already doing that before :-/